src/mokka/src/main/resources/templates/index.html

```diff
@@ -20,27 +21,19 @@
 		<md-toolbar>
 	      <div class="md-toolbar-tools">
 	        <h2>
-	          <span>Mokka</span>
-	          <span th:if="${path == 'login'}">&nbsp;>&nbsp;Login</span>
-	          <span th:if="${path == 'log'}">&nbsp;>&nbsp;Logs</span>
-	          <span th:if="${path == 'configuration'}">&nbsp;>&nbsp;Configuration</span>
+	          <div id="logoMokka"></div>
 	        </h2>
 
 			<md-menu sec:authorize="hasRole('ROLE_USER')">
```

Comment:
I do not think we need md-menu and md-content anymore as there is no real menu here.
Also buttons may be better aligned vertically (center).

I propose to use `<section layout="row" layout-sm="column" layout-align="center center" layout-wrap sec:authorize="hasRole('ROLE_USER')">` should fix both issues